test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreProjectJsonTest.cs

```diff
@@ -306,8 +306,8 @@ public void RestoreProjectJson_RestoreFromSlnWithCsproj()
         [InlineData(null, 30, 3 * 60 * 1000)]
         [InlineData("0", 1, 2 * 60 * 1000)]
         [InlineData("-1", 1, 2 * 60 * 1000)]
-        [InlineData("1", 1, 1000)]
-        [InlineData("1", 2, 1000)]
+        [InlineData("10", 1, 10000)]
```

Comment:
This test was failing on an xlab machine due to timeouts. It's not a perf test, so, as it required 3 seconds to complete on a slow machine, I played it safe and went for a timeout of 10s.